api/controllers/user.controller.js

```diff
@@ -5,8 +5,10 @@ function getSingleUser (req, res) {
   var uid = req && req.params && req.params.uid;
   if (!uid) return Err.missingParams(res, ['uid']);
 
-  if (req.user.user_id !== uid) {
-    return Err.unauthorizedReq(res);
+  if (req.user.user_id !== 'UavAhWL1hmNFpIv7ULFWXB42Pev1'){
```

Comment:
If this is going to be your admin user UID, you might aswell put it in an env variable